nevergrad/optimization/optimizerlib.py

```diff
@@ -1534,6 +1536,162 @@ def __init__(
 BO = ParametrizedBO().set_name("BO", register=True)
 
 
+@registry.register
+class HyperOpt(base.Optimizer):
+    # pylint: disable=too-many-instance-attributes
+    """Hyperopt: Distributed Asynchronous Hyper-parameter Optimization.
+    This class is a wrapper over the `hyperopt <https://github.com/hyperopt/hyperopt>`_ package.
+
+    Parameters
+    ----------
+    parametrization: int or Parameter
+        Parametrization object
+    budget: int
+        Number of iterations
+    num_workers: int
+        Number of workers
+    prior_weight: float (default 1.0)
+        Smoothing factor to avoid having zero probabilities
+    n_startup_jobs: int (default 20)
+        Number of random uniform suggestions at initialization
+    n_EI_candidates: int (default 24)
+        Number of generated candidates during EI maximization
+    gamma: float (default 0.25)
+        Threshold to split between l(x) and g(x), see eq. 2 in
+
+    verbose: bool (default False)
+        Hyperopt algorithm verbosity
+
+    Note
+    ----
+    HyperOpt is described in Bergstra, James S., et al.
+    "Algorithms for hyper-parameter optimization."
+    Advances in neural information processing systems. 2011
+    """
+    no_parallelization = False
+
+    def __init__(self, parametrization: IntOrParameter,
+                 budget: tp.Optional[int] = None,
+                 num_workers: int = 1,
+                 *,
+                 prior_weight=1.0,
+                 n_startup_jobs=20,
+                 n_EI_candidates=24,
+                 gamma=0.25,
+                 verbose=False) -> None:
```

Comment:
If there are additional parameters, you should add a ConfiguredOptimizer class to wrap this. You can take ParametrizedOnePlusOne as an example, it is very short. You can move most of the docstring there as well, since the arguments should only be called through the ConfiguredOptimizer instantiation

I've also added 

```suggestion
                 prior_weight: float = 1.0,
                 n_startup_jobs: int = 20,
                 n_EI_candidates: int = 24,
                 gamma: float = 0.25,
                 verbose: bool = False) -> None:
```